controllers/rhmi/bootstrapReconciler.go

```diff
@@ -129,6 +130,12 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 		return phase, errors.Wrap(err, "failed to reconcile github oauth secrets")
 	}
 
+	phase, err = r.reconcileAddonManagedApiServiceParameters(ctx, serverClient)
+	if err != nil || phase != integreatlyv1alpha1.PhaseCompleted {
+		events.HandleError(r.recorder, installation, phase, "Failed to reconcile addon-managed-api-service-parameters", err)
```

Comment:
this message might be misleading imo - the secret could be also called `addon-managed-api-service-internal-parameters` and there is a chance that it will be also different for MT RHOAM in the future. Can we just say that we `failed to reconcile the addon parameters secret`? :thinking:

Reply:
guess I want to cover the only use case that CS-SRE will care about customer clusters, the SOP covers if its a different addon name https://github.com/RHCloudServices/integreatly-help/pull/828/files#diff-a062bd65551cf4a6220379a4d0f187ace6396aacf1711372005cfcf277086a16R25 guess there is no harm in changing it.